book-a-reading-room-visit.api/Controllers/BookingController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using book_a_reading_room_visit.api.Models;
+using book_a_reading_room_visit.api.Service;
+using book_a_reading_room_visit.domain;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+
+namespace book_a_reading_room_visit.api.Controllers
+{
+    [Route("book-a-visit-api/[controller]")]
+    [ApiController]
+    public class BookingController : ControllerBase
+    {
+        private IBookingService _bookingService;
+
+        public BookingController(IBookingService bookingService)
+        {
+            _bookingService = bookingService;
+        }
+
+        [HttpGet("search")]
+        public async Task<ActionResult<Booking>> GetBookings([FromQuery]BookingSearchModel bs)
```

Comment:
- rename the parameter
- do we need the [FromQuery] for a GET method?

Reply:
OK have renamed paramter.  The [FromQuery] is required for the binding to work (otherwise 415 error and not rendered correctly in Swagger)

Reply:
Should we also validate the Booking Reference e.g. will it always be a letter followed by 6 numbers?

Reply:
.. or a 10 digit number. And is a Reader Ticket always 7 digits?